src/SignalR/server/SignalR/src/SignalRDependencyInjectionExtensions.cs

```diff
@@ -48,10 +58,15 @@ public static ISignalRServerBuilder AddSignalR(this IServiceCollection services)
         /// Adds SignalR services to the specified <see cref="IServiceCollection" />.
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
-        /// <param name="configure">An <see cref="Action{MvcOptions}"/> to configure the provided <see cref="HubOptions"/>.</param>
+        /// <param name="configure">An <see cref="Action{HubOptions}"/> to configure the provided <see cref="HubOptions"/>.</param>
```

Comment:
Good catch. While you're here can you also update the param doc for the action on `AddHubOptions` ?

Reply:
It doesn't have the same mistake. Referencing generics in xmldoc is difficult.